core/src/SYCL/Kokkos_SYCL_Space.cpp

```diff
@@ -217,14 +217,15 @@ void SYCLSharedUSMSpace::deallocate(const char* arg_label,
                   Kokkos::Tools::make_space_handle(name()));
 }
 
-void SYCLDeviceUSMSpace::impl_access_error() {
+KOKKOS_DEPRECATED void SYCLDeviceUSMSpace::impl_access_error() {
```

Comment:
Why not just remove?  The CUDA one has been around for while.  This one has not been part of any release (?)

Reply:
My initial thinking was to keep some kind of symmetry with the other backends but I now think it's fair to remove it. Let me do that.

Reply:
You forgot the definitions

Reply:
Fixed.